README.md

```diff
@@ -6,6 +6,8 @@
 
 This is an AEM Commerce connector for Magento and GraphQL, that provides some integration of Magento products and categories in the AEM Commerce console, and some authoring features like product and category pickers.
 
+**Attention**: This project is for AEM 6.4/6.5 projects only and works with AEM quickstart. It is not intended to be used for AEM as a Cloud Service or with the AEM SDK.
```

Comment:
I'd write "AEM Cloud SDK" to make it more clear what this is.

Reply:
Yes, changed.